operations/addresses.md

```diff
@@ -211,48 +226,49 @@ Updates one or more existing addresses in the system, assigned to specified acco
-| `Line2` | [String update value](_objects.md#string-update-value) | optional | Second line of the address. |
-| `City` | [String update value](_objects.md#string-update-value) | optional | The city. |
-| `PostalCode` | [String update value](_objects.md#string-update-value) | optional | Postal code. |
-| `CountryCode` | [String update value](_objects.md#string-update-value) | optional | ISO 3166-1 code of the [Country](countries.md#country). |
-| `CountrySubdivisionCode` | [String update value](_objects.md#string-update-value) | optional | ISO 3166-2 code of the administrative division, e.g. `DE-BW`. |
+| `Line1` | [String update value](_objects.md#string-update-value) | optional, max length 255 characters | First line of the address. |
+| `Line2` | [String update value](_objects.md#string-update-value) | optional, max length 255 characters | Second line of the address. |
+| `City` | [String update value](_objects.md#string-update-value) | optional, max length 255 characters | The city. |
+| `PostalCode` | [String update value](_objects.md#string-update-value) | optional, max length 255 characters | Postal code. |
+| `CountryCode` | [String update value](_objects.md#string-update-value) | optional, max length 8 characters | ISO 3166-1 code of the Country. |
+| `CountrySubdivisionCode` | [String update value](_objects.md#string-update-value) | optional, max length 8 characters | ISO 3166-2 code of the administrative division, e.g. DE-BW. |
+| ~~`AccountId`~~ | ~~string~~ | ~~required~~ | **Deprecated!** The value is ignored.|
 
 ### Response
 
 ```javascript
 {
-    "Addresses": [
-        {
-            "Id": "fc7b2df3-de66-48a6-907d-af4600ecd892",
-            "AccountId": "3db2c989-7d95-42b4-a502-a9f246db1634",
-            "AccountType": "Customer",
-            "Line1": "I.P. Pavlova 5",
-            "Line2": null,
-            "City": "Prague",
-            "PostalCode": "12000",
-            "CountryCode": "CZ",
-            "CountrySubdivisionCode": null,
-            "Latitude": null,
-            "Longitude": null,
-            "UpdatedUtc": "2023-11-29T14:49:29.982Z",
-            "IsActive": true
-        }
-    ]
+  "Addresses": [
+    {
+      "Id": "fc7b2df3-de66-48a6-907d-af4600ecd892",
+      "AccountId": "3db2c989-7d95-42b4-a502-a9f246db1634",
+      "AccountType": "Customer",
+      "Line1": "I.P. Pavlova 5",
+      "Line2": null,
+      "City": "Prague",
+      "PostalCode": "12000",
+      "CountryCode": "CZ",
+      "CountrySubdivisionCode": null,
+      "Latitude": null,
+      "Longitude": null,
+      "UpdatedUtc": "2023-11-29T14:49:29.982Z",
+      "IsActive": true
+    }
+  ]
 }
 ```
 
 | Property | Type | Contract | Description |
 | :-- | :-- | :-- | :-- |
-| `Addresses` | array of [Account address](#account-address) | required | Updated addresses. |
+| `Addresses` | array of [Account address](addresses.md#account-address) | optional | The collection of Account addresses, containing address and account information. |
```

Comment:
```suggestion
| `Addresses` | array of [Account address](addresses.md#account-address) | require | The collection of Account addresses, containing address and account information. |
```
Same here as on comments above.

Reply:
line 1 now 1023, lines 2: 255 as in the database